config/locales/forms.yml

```diff
@@ -610,17 +610,22 @@ en:
         completed_period_hint: Early career teachers (ECTs) have to complete a 2 year induction period. This used to be 1 year for newly qualified teachers (NQTs).
         teacher_reference_number_hint: Schools will only be able to see your TRN if you apply for a role. You can %{link} if you do not know it (for example, if you're a trainee teacher) or have forgotten it. Your TRN is 7 digits long.
         trn_link_text: find your TRN
-
-
       jobseekers_training_and_cpd_form:
         name: Name of course or training
         provider: Training provider
         grade: Grade (optional)
         year_awarded: Date completed
         year_completed_hint: For example, 1998
         course_length: Course length
-
-
+      jobseekers_professional_body_membership_form:
```

Comment:
I have been breaking these changes out into individual yml files - personally these files are scary big and really hard to work with - which section is which? (hint, legend and label). I think we can slowly tame them, but I'm finding it much nicer (and much quicker) than hacking 3  1k line translation files